src/scss/_righttoc.scss

```diff
@@ -4,7 +4,7 @@
 
 .bd-toc {
     padding: 0px !important;
-    overflow-y: visible;
+    overflow-y: auto;
```

Comment:
auto should be the default, so omitting this rule might also work. :)

Reply:
I kinda like it from an "explicit is better than implicit" standpoint...but could remove if you think it's not necessary

Reply:
I'm fine either way! :)